### ISSUE TITLE
115+ not working on M1 MacBook

### ISSUE THREAD

**Issue by OP-Klaus** [giggio/node-chromedriver#428]
Hi, thank you for this project, I have been using it for a long time and really appreciate it!

From v115 it seems that M1 MacBooks (ARM) can't find the Chrome binary when using WebDriverIO. A few people on my team on different machines have reproduced it too.

The reason I am posting here instead of on the WebDriverIO tracker is that rolling back `node-chromedriver` to v114 fixes it, even with Chrome v115 installed.

The error as logged by WebDriverIO is:
```
[wdio] [0-0] Error: Failed to create session.
[wdio] unknown error: cannot find Chrome binary
```

Any help would be awesome

**Comment by giggio** [comment]
This is an issue with Chromedriver, and was already patched and released. I just integrated a PR that should fix it and I'll release it in a moment as version `115.0.1`. If it still persists after that please let me know.
Closed on #427.

**Comment by OP-Klaus** [comment]
@giggio thank you for the speed on this, I have confirmed the fix works 🎉
